Makefile

```diff
@@ -1,12 +1,16 @@
-INTEGRATION     := docker
-BINARY_NAME      = nri-$(INTEGRATION)
-SRC_DIR          = ./src/
-VALIDATE_DEPS    = golang.org/x/lint/golint
-INTEGRATIONS_DIR = /var/db/newrelic-infra/newrelic-integrations/
-CONFIG_DIR       = /etc/newrelic-infra/integrations.d
-GO_FILES        := ./src/
-GOOS             = GOOS=linux
-GO               = $(GOOS) go
+INTEGRATION     	:= docker
+BINARY_NAME      	= nri-$(INTEGRATION)
+SRC_DIR          	= ./src/
+VALIDATE_DEPS    	= golang.org/x/lint/golint
+INTEGRATIONS_DIR 	= /var/db/newrelic-infra/newrelic-integrations/
+CONFIG_DIR       	= /etc/newrelic-infra/integrations.d
+GO_FILES        	:= ./src/
+GOOS             	= GOOS=linux
+GO               	= $(GOOS) go
+GO_VERSION 			?= $(shell grep '^go ' go.mod | awk '{print $$2}')
+UBUNTU_VERSION 		?= "16.04"
+BUILDER_IMAGE_TAG 	?= "latest"
```

Comment:
if no one is setting this I would not make it configurable and just place them in the BUILDER_IMAGE (that is already configurable)

Reply:
Makes sense. Done. Thanks for suggesting.